test/generation.test.js

```diff
@@ -16,8 +18,51 @@ describe('Generation tests', () => {
         httpVersion: '2',
     });
 
+    test('Generates unordered headers', () => {
+        const result = headerGenerator.getRawHeadersAndOrder();
+
+        expect(result).toBeTruthy();
+        expect(result.generatedSample).toBeTruthy();
+        expect(Array.isArray(result.order)).toBe(true);
+    });
+
     test('Generates headers', () => {
-        expect(headerGenerator.getHeaders()).toBeTruthy();
+        const headers = headerGenerator.getHeaders();
+
+        let userAgent;
+        for (const [header, value] of Object.entries(headers)) {
+            if (header.toLowerCase() === 'user-agent') {
+                userAgent = value;
+                break;
+            }
+        }
+
+        let browser;
+        if (userAgent.includes('Firefox')) {
+            browser = 'firefox';
+        } else if (userAgent.includes('Chrome')) {
+            browser = 'chrome';
+        } else {
+            browser = 'safari';
+        }
+
+        expect(typeof headers).toBe('object');
+
+        const order = headersOrder[browser];
+
+        let index = -1;
+        for (const header of Object.keys(headers)) {
+            const newIndex = order.indexOf(header);
+
+            const log = `${userAgent}\n${browser}\n${inspect(order)}\n${inspect(headers)}`;
```

Comment:
The `inspect` function from the `util` module nicely formats objects and arrays so it's easier to read them in logs :)